drivers/wifi/nrf_wifi/src/wpa_supp_if.c

```diff
@@ -1595,46 +1599,57 @@ void nrf_wifi_wpa_supp_event_get_wiphy(void *if_priv,
 
 	vif_ctx_zep = if_priv;
 	rpu_ctx_zep = vif_ctx_zep->rpu_ctx_zep;
+	vif_ctx_cnt = nrf_wifi_fmac_get_num_vifs(rpu_ctx_zep->rpu_ctx);
 
 	memset(&band, 0, sizeof(band));
 
-	for (int i = 0; i < NRF_WIFI_EVENT_GET_WIPHY_NUM_BANDS; i++) {
-		if (nrf_wifi_parse_sband(&wiphy_info->sband[i], &band) != WLAN_STATUS_SUCCESS) {
+	for (int idx = 0; idx < vif_ctx_cnt; ++idx) {
+		vif_ctx_zep = nrf_wifi_get_vif_ctx_by_idx(idx);
+		if (vif_ctx_zep == NULL) {
 			continue;
 		}
-		if (vif_ctx_zep->supp_drv_if_ctx && vif_ctx_zep->supp_callbk_fns.get_wiphy_res) {
-			vif_ctx_zep->supp_callbk_fns.get_wiphy_res(vif_ctx_zep->supp_drv_if_ctx,
-									&band);
+
+		for (int i = 0; i < NRF_WIFI_EVENT_GET_WIPHY_NUM_BANDS; i++) {
+			if (nrf_wifi_parse_sband(&wiphy_info->sband[i], &band) !=
+				WLAN_STATUS_SUCCESS) {
+				continue;
+			}
+			if (vif_ctx_zep->supp_drv_if_ctx &&
+				vif_ctx_zep->supp_callbk_fns.get_wiphy_res) {
+				vif_ctx_zep->supp_callbk_fns.get_wiphy_res(
+					vif_ctx_zep->supp_drv_if_ctx, &band);
+			}
 		}
-	}
 
-	if ((wiphy_info->params_valid & NRF_WIFI_GET_WIPHY_VALID_EXTENDED_CAPABILITIES) &&
-	    rpu_ctx_zep->extended_capa == NULL) {
-		/* To avoid overflowing the 100 column limit */
-		unsigned char ec_len = wiphy_info->extended_capabilities_len;
+		if ((wiphy_info->params_valid & NRF_WIFI_GET_WIPHY_VALID_EXTENDED_CAPABILITIES) &&
+			rpu_ctx_zep->extended_capa == NULL) {
+			/* To avoid overflowing the 100 column limit */
+			unsigned char ec_len = wiphy_info->extended_capabilities_len;
 
-		rpu_ctx_zep->extended_capa = nrf_wifi_osal_mem_alloc(ec_len);
+			rpu_ctx_zep->extended_capa = nrf_wifi_osal_mem_alloc(ec_len);
 
-		if (rpu_ctx_zep->extended_capa) {
-			memcpy(rpu_ctx_zep->extended_capa, wiphy_info->extended_capabilities,
-			       ec_len);
-		}
+			if (rpu_ctx_zep->extended_capa) {
+				memcpy(rpu_ctx_zep->extended_capa,
+					wiphy_info->extended_capabilities, ec_len);
+			}
 
-		rpu_ctx_zep->extended_capa_mask = nrf_wifi_osal_mem_alloc(ec_len);
+			rpu_ctx_zep->extended_capa_mask = nrf_wifi_osal_mem_alloc(ec_len);
```

Comment:
Unnecessary change, extra tab in the line

I am also seeing lot of indentation issues in this PR. For example 
```
				memcpy(rpu_ctx_zep->extended_capa,
					wiphy_info->extended_capabilities, ec_len);
```
above, the second line should align with `rpu_ctx->...`, now it is 1 char off (at least my GH review screen tells that).
Are you able to adjust your editor to align these properly automatically?